bin/ch-builder2squash

```diff
@@ -36,18 +36,25 @@ if [ "$#" -lt 2 ]; then
 fi
 
 builder_choose
-echo "builder: ${CH_BUILDER}"
 
 image=$1
 outdir=$2
 shift 2
 
+if [ ! -e "$outdir" ]; then
+    echo "output directory: ${outdir} does not exist" 1>&2
+    exit 1
+fi
+
 # mktemp is used for intermediate files to avoid heavy metadata loads on
 # certain filesystems
 temp=$(mktemp -d --tmpdir ch-builder2squash.XXXXXX)
 # Get image as a directory
 "${ch_bin}/ch-builder2tar" -b "$CH_BUILDER" --nocompress "$image" "$temp"
+
+image=$(echo "$image" | sed 's/\//./g')
```

Comment:
Is there any way to share the same naming code as wherever else does this substitution?

Reply:
As in assign it the variable `tar` like in `ch-builder2tar`?

That is certainly possible. We would still need this call to identify the output of `ch-builder2tar`. So I could assign a variable to the tarball in addition to the image directory it will produce. 

Unless I'm wildly misunderstanding what you're asking me to do.

Reply:
What I'm thinking is that somewhere we already have code that translates a Docker image tag to something suitable for a filesystem, so I propose find that code and put it in a function in `base.sh`.